src/pages/Login.tsx

```diff
@@ -193,7 +194,27 @@ const Login = () => {
         </div>
         <Button
           onClick={() => {
-            setUser({ id: email, password: password });
+            if (!email.endsWith('@kakao.com')) {
+              toast.warn('@kakao.com 이메일 주소만 가능합니다.', {
+                position: 'bottom-center',
+                autoClose: 5000,
+                hideProgressBar: false,
+                closeOnClick: false,
+                pauseOnHover: true,
+                draggable: true,
+                progress: undefined,
+                theme: 'dark',
+                transition: Bounce,
+                style: {
+                  width: '25rem',
+                  color: 'black',
+                  backgroundColor: 'white',
+                },
+              });
+
+              return;
+            }
+            setUser({ email: email, password: password });
```

Comment:
요런식으로 줄여서 사용할 수도 있어용! 
```suggestion
            setUser({ email, password });
```